### ISSUE TITLE
2020 general - Senate

### ISSUE THREAD

**Issue by jmillermi** [openelections/openelections-data-ne#33]
Results for Senate in 2020 seem to actually be results for US House District 3?

https://ballotpedia.org/Nebraska%27s_3rd_Congressional_District_election,_2020

(There are no results in the file for U.S. Senate or the other 2 House seats)

**Comment by dwillis** [comment]
@jmillermi you're right; I've fixed the U.S. House 3 results to show the correct office and district and will see about getting the remainder of the missing results.
